sdk/sdkmodule/unpack_test.go

```diff
@@ -26,5 +33,38 @@ func TestUnpack(t *testing.T) {
 	}, "i", &i, "s=", &s)) {
 		assert.Equal(t, 64, i)
 		assert.Equal(t, "meow", s)
+		assert.Zero(t, st)
 	}
 }
+
+func TestUnpackFlat(t *testing.T) {
```

Comment:
could you/we convert those 2 tests to be parametrized? and cover more cases? (e.g. args, kwargs, missing kwars, optional, more types, etc?)

Reply:
done